server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -480,6 +480,9 @@ func (s *BuildBuddyServer) GetApiKeys(ctx context.Context, req *akpb.GetApiKeysR
 			VisibleToDevelopers: k.VisibleToDevelopers,
 		})
 	}
+	if al := s.env.GetAuditLog(); al != nil {
+		al.Log(ctx, auditlog.GroupAPIKeyResourceID(""), auditlog.ListAPIKeys, req)
```

Comment:
for my own understanding - are we using a convention that an empty resource ID always represents the list of all the resources? (if so, are there other resource types where that convention would not work due to empty string being valid, and maybe lead to confusion?)

Reply:
hm actually since the operation name is ListAPIKeys I think this is fine.

Reply:
This is basically setting resource_type = API_KEY w/o setting an id. This will allow the "list API keys" operations to be returned if the user queries all logs for API Key resources.